server/utils/timezones.go

```diff
@@ -0,0 +1,610 @@
+package utils
+
+var windowsToIanaMap = map[string]string{}
+
+func ConvertWindowsTimezoneToIANA(winTZ string) string {
+	if len(windowsToIanaMap) == 0 {
+		populateConversionMap()
+	}
+
+	tz, ok := windowsToIanaMap[winTZ]
+	if ok {
+		return tz
+	}
+
+	return ""
+}
+
+func populateConversionMap() {
```

Comment:
Seems unnecessary since a few substitutions can make this into an idiomatic go map.